lib/main.dart

```diff
@@ -11,14 +11,26 @@ import 'package:mobile_nebula/screens/MainScreen.dart';
 import 'package:mobile_nebula/screens/EnrollmentScreen.dart';
 import 'package:mobile_nebula/services/settings.dart';
 import 'package:flutter_web_plugins/url_strategy.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
 
-//TODO: EventChannel might be better than the stream controller we are using now
-
-void main() {
+Future<void> main() async {
   usePathUrlStrategy();
-  runApp(Main());
+  await SentryFlutter.init(
```

Comment:
Is it possible to add sourcemaps here? Does Dart have sourcemaps / debug symbols? It'd be great to improve the stacktraces in Sentry if possible

Reply:
According to https://docs.sentry.io/platforms/flutter/upload-debug/#when-to-upload, there's no "flutter sourcemaps" that we need to upload, but we may want to consider uploading debug symobols for the native android/ios code.  I think that can be done in a followup.